ledger/participant-state/kvutils/tools/src/main/scala/com/daml/ledger/participant/state/kvutils/tools/Print.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.tools
+
+import scala.io.AnsiColor
+
+object Print {
```

Comment:
Having such a utility class is great for colored printing. However, I'd prefer something less intrusive via e.g. implicits, similar to the solution here:
https://yobriefca.se/blog/2014/01/06/convenient-colouring-of-console-output-in-scala/
The only tweak that would be essential is adding a `$RESET` at the end of the printed text automatically so that you don't forget it (and everything will be printed in the color you postfix the string with). What do you think?

Reply:
Sounds like a cool idea to me. I'll change it.

Reply:
Done.